### ISSUE TITLE
[Feature] Consider publishing a normalised pre-trained  model

### ISSUE THREAD

**Issue by alessandrofelder** [brainglobe/cellfinder#617]
**Is your feature request related to a problem? Please describe.**
The cellfinder pre-trained does not come with statistics, which is expected to worsen classification in some cases, and increase amount of fine-tuning needed.

**Describe the solution you'd like**
Create a new pre-trained model based on the same training data as the current pre-trained model, but with the normalisation enabled.

**Additional context**
Follow-on from https://github.com/brainglobe/cellfinder/pull/588

**Comment by adamltyson** [comment]
Also linked to https://github.com/brainglobe/cellfinder/issues/450#issuecomment-4498372359

**Comment by matham** [comment]
The best way to get the required statistics for the existing training set without re-exporting all the cuboids is to do the following.

For each dataset in the original training dataset (i.e. each yaml/folder in the training set):
1. Open the original tiff (not extracted cubes) from which these cubes came from.
2. Label one cell (and one non-cell if curation needs that).
3. Save the cubes.
4. This will have generated a yaml file for this dataset. Open the yaml, copy the attributes with the statistics and paste it into the old yaml files for the cells and non-cells that came from this dataset.

**Comment by adamltyson** [comment]
> Open the original tiff (not extracted cubes) from which these cubes came from.

I don't think we have these unfortunately.
